src/ui/interface.js

```diff
@@ -1107,11 +1109,11 @@ export class UI {
 			this.abilitiesButtons.forEach((btn) => {
 				let ab = creature.abilities[btn.abilityId];
 				btn.css.normal = {
-					"background-image": "url('../units/abilities/" + creature.name + " " + btn.abilityId + ".svg')"
+					"background-image": `url(${getUrl("units/abilities/" + creature.name + " " + btn.abilityId)})`
```

Comment:
You removed both `'` from this line, intentional? All the others still have the single quotes inside the `url()` call.

Reply:
Yeah it's a mistake. It's not valid CSS